scripts/tests/run_tv_casting_test.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env -S python3 -B
+
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import subprocess
+import sys
+import time
+
+import click
+
+# Configure logging format.
+logging.basicConfig(level=logging.DEBUG, format='%(levelname)s - %(message)s')
+
+LINUX_TV_APP_LOGS = './scripts/tests/Linux-tv-app-logs.txt'
```

Comment:
This seems to access paths inside scripts ... those are source files. you should not dirty the source tree, even teporarely. Can we use temporary files instead?

Reply:
Sure, I revised the code to use a temporary directory to store the log files.